crowbar_framework/vendor/assets/stylesheets/jquery/ledUpdate.scss

```diff
@@ -48,3 +48,27 @@
     background: image-url("jquery/ledUpdate/yellow-green-spin.png");
   }
 }
+
+.status-border-unready {
+  border-left: 7px solid #F8E71C!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>Color literals like `#F8E71C` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F8E71C` should be written as `#f8e71c`<br>!important should not be used<br>Properties should be ordered animation-duration, animation-iteration-count, animation-name, border-left